### ISSUE TITLE
safetensors file formats are not recognized

### ISSUE THREAD

**Issue by Kilvoctu** [RupertAvery/DiffusionToolkit#12]
https://github.com/RupertAvery/DiffusionToolkit/issues/12#issuecomment-1343025101

~~Hello, I've observed an issue in which the toolkit is not displaying all models in the models list when there are folders in the "models" folder. This will cause an issue where many images will display "Not found" under "Model Name" when inspecting its png info.~~

~~I have model root set correctly to where all my models are located, and I've tried the "Rescan Folders" button.~~
~~`[path to webui]\models\Stable-diffusion`~~

~~My model root folder is formatted sort of like this~~

 - ~~1 _(subfolder)_~~
   - ~~modelD _(ckpt file)_~~
 - ~~2 _(subfolder)_~~
   - ~~modelE _(ckpt file)_~~
 - ~~3 _(subfolder)_~~
   - ~~modelF _(ckpt file)_~~
 - ~~modelA _(ckpt file)_~~
 - ~~modelB _(ckpt file)_~~
 - ~~modelC _(ckpt file)_~~

~~In this condition, the toolkit is displaying only "modelF", in subfolder "3". The root folder and other folders are not recongized.~~

**Comment by RupertAvery** [comment]
Hi, I have a similar setup, with several subfolders. It seems to work for me. Is there any other information you can give?

**Comment by Kilvoctu** [comment]
Red herring, I think, got me confused. I should have just provided my actual file/folder environment instead of creating a simplified example.

The way I set up my folders, my last folder has all my ckpt files. Everything else was converted to safetensors file format. I tested by shuffling some files around to confirm, and indeed the application just doesn't recognize safetensors format.
Models tab is only showing ckpt files, no safetensors

Sorry about the poor troubleshooting, I'll update the title, too.
